docs/src/main/paradox/sqs.md

```diff
@@ -117,12 +117,15 @@ Options:
 
 ### Message processing with acknowledgement
 
-`SqsAckSink` provides possibility to acknowledge (delete) or requeue a message.
+`SqsAckSink` provides possibility to acknowledge (delete), ignore, or postpone a message.
 
 Your flow must decide which action to take and push it with message:
 
- - `Ack` - delete message from the queue
- - `RequeueWithDelay(delaySeconds: Int)` - schedule a retry
+ - `Delete` - delete message from the queue
+ - `Ignore` - ignore the message and let it reappear in the queue after visibility timeout
+ - `ChangeMessageVisibility(visibility: Int)` - can be used to postpone a message, or make
```

Comment:
I think parameter should be called `visibilityTimeout` as it is referred to like that in the AWS docs.